content/aips/AIP-add-XSGD.md

```diff
@@ -0,0 +1,100 @@
+---
+aip: <to be assigned>
+title: Add XSGD support as Collateral
+status: WIP
+author: Yap Jun Hao (@junhaoyap), Chung Chao (@chung-chao)
+shortDescription: Add Support for the Digital Singapore Dollar XSGD
+discussions: https://governance.aave.com/t/arc-add-xsgd-support-as-collateral/4260
+created: 2021-06-09
+updated: 2021-06-09
+# requires (*optional): <AIP number(s)>
```

Comment:
We won't need this, right?

Reply:
@junhaoyap Did you mean this four line or whole the table? I take this as an example: https://github.com/aave/aip/pull/66, which this 10 lines are coming from AIP template as well.

No specification from Aave that we should put this. I can ask them directly.